build/root/.bazelrc

```diff
@@ -39,3 +39,32 @@ build:cross:linux_arm --config=repo_infra_crosstool --platforms=@io_bazel_rules_
 build:cross:linux_arm64 --config=repo_infra_crosstool --platforms=@io_bazel_rules_go//go/toolchain:linux_arm64 --cpu=arm64
 build:cross:linux_ppc64le --config=repo_infra_crosstool --platforms=@io_bazel_rules_go//go/toolchain:linux_ppc64le --cpu=ppc64le
 build:cross:linux_s390x --config=repo_infra_crosstool --platforms=@io_bazel_rules_go//go/toolchain:linux_s390x --cpu=s390x
+
+# --config=remote-cache enables a remote bazel cache
+# Note needs a --remote_instance_name=projects/PROJ/instances/default_instance flag
+build:remote-cache --remote_cache=remotebuildexecution.googleapis.com
+build:remote-cache --tls_enabled=true
+build:remote-cache --remote_timeout=3600
+build:remote-cache --auth_enabled=true
```

Comment:
Why do we only want --auth_enabled and --tls_enabled on remote-cache?

Reply:
It's a composition: `build:remote --config=remote-cache`

Reply:
Sure, but the question is "why aren't --auth_enabled and --tls_enabled on --config=remote rather than the composite?"

Reply:
They are. See L52. The remote config includes everything in remote-cache (not the reverse).

Reply:
Ah thanks.